eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/microfragments/eventdetails/fragments/views/EventHeaderDateTimeView.java

```diff
@@ -46,6 +46,8 @@ public EventHeaderDateTimeView(SchedjoulesDetailsHeaderBinding binding)
     @Override
     public void update(Event event)
     {
+        // TODO: go back to this version once the API stops to return "estimated" end times:
+        // CharSequence rangeCharSequence = new DateTimeRange(event).value(mDateTimeViewCollapsed.getContext());
```

Comment:
The 'interoperability' of `DateTimeRange` and `LongDateAndTimeNoYear` is not clear for me here, do they surely only differ in showing end time as well. But it's probably not important at this point, I guess we can double check when we switch back to showing the end times as the TODO say.
We could possibly create an `EventDateTime` that delegates based on whether duration is present or not, if we need to use 2 formatters.